quizcon/templates/main/quiz_form_edit.html

```diff
@@ -18,6 +18,14 @@
         jQuery("#id_show_answers_date").attr("disabled", true)
     });
 </script>
+{% if quiz.quizsubmission_set.count %}
```

Comment:
The modal popped up twice for me when navigating from the Quizzes dashboard, which was a bit irritating. I wonder if having the warning on the Quiz Detail page is enough?

Also thinking that we might want to disable some edit options that could be problematic after a submission happens, e.g. Consequences.